src/test/java/Homework16.java

```diff
@@ -0,0 +1,44 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class Homework16 {
+
+    @Test
+    public static void Navigation() throws InterruptedException {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(30));
+
+        String url = "https://bbb.testpro.io/";
+        String url2 = "https://bbb.testpro.io/registration.php";
```

Comment:
This variable should be renamed to something more descriptive , like registrationUrl

**I recommend to use in the variable naming:**
-Follow code convention
-Use comprehensive names
-1 and 0 harcoded